src/core/types/function.js

```diff
@@ -36,7 +36,7 @@ export function once(fn) {
   let callback = fn;
   return (...args) => {
```

Comment:
Ugh, this shouldn't have been here to start with.

Reply:
Yeah I'd been wondering about this, and if it caused the iterator polyfill to be included regardless. Should I rewrite to use `arguments`?